Makefile

```diff
@@ -322,29 +323,6 @@ generate-go-conversions-kubeadm-control-plane: $(CONVERSION_GEN) ## Generate con
 		--output-file-base=zz_generated.conversion $(CONVERSION_GEN_OUTPUT_BASE) \
 		--go-header-file=./hack/boilerplate/boilerplate.generatego.txt
 
```

Comment:
Just fyi. I'll re-introduce those targets in my catalog PR with correct paths (but fine to drop here)